GPU/Common/PresentationCommon.h

```diff
@@ -127,6 +127,7 @@ class PresentationCommon {
 
 	void GetCardboardSettings(CardboardSettings *cardboardSettings) const;
 	void CalculatePostShaderUniforms(int bufferWidth, int bufferHeight, int targetWidth, int targetHeight, const ShaderInfo *shaderInfo, PostShaderUniforms *uniforms) const;
+	int CalculateZoom() const;
```

Comment:
nit: Refactoring that didn't happen?

-[Unknown]

Reply:
indeed, didn't work out. removing.